src/compiler/checker.ts

```diff
@@ -42869,149 +42896,217 @@ namespace ts {
+                    if (!fluentCache.has(target)) {
+                        fluentCache.set(target, new Map());
+                    }
+                    const map = fluentCache.get(target)!;
+                    map.set(name, {
+                        patched: getEtsFluentSymbolForFunctionDeclaration(name, declaration),
+                        exportName: declaration.name.escapedText.toString(),
+                        definition: file
+                    });
+                }
+            }
+        }
+        function tryCacheEtsGetterFunction(file: SourceFile, declaration: FunctionDeclaration) {
+            if(declaration.name && isIdentifier(declaration.name)) {
+                const getterTag = collectEtsGetterTags(declaration)[0];
+                if (getterTag) {
+                    const [, target, name] = getterTag.comment.split(" ");
+                    if (!getterCache.has(target)) {
+                        getterCache.set(target, new Map());
+                    }
+                    const map = getterCache.get(target)!;
+                    map.set(name, {
+                        patched: getEtsGetterSymbolForFunctionDeclaration(name, declaration),
+                        exportName: declaration.name.escapedText.toString(),
+                        definition: file
+                    });
+                }
+            }
+        }
+        function tryCacheEtsStaticFunction(file: SourceFile, declaration: FunctionDeclaration) {
+            if(declaration.name && isIdentifier(declaration.name)) {
+                const staticTag = collectEtsStaticTags(declaration)[0];
+                if (staticTag) {
+                    const [, target, name] = staticTag.comment.split(" ");
+                    if (!staticCache.has(target)) {
+                        staticCache.set(target, new Map());
+                    }
+                    const map = staticCache.get(target)!;
+                    map.set(name, {
+                        patched: getEtsStaticSymbolForFunctionDeclaration(name, declaration),
+                        exportName: declaration.name.escapedText.toString(),
+                        definition: file
+                    });
+                }
+            }
+        }
+        function tryCacheEtsUnifyFunction(declaration: FunctionDeclaration) {
+            if(declaration.name && isIdentifier(declaration.name)) {
+                const unifyTag = collectEtsUnifyTags(declaration)[0];
+                if (unifyTag) {
+                    const [, target] = unifyTag.comment.split(" ");
+                    identityCache.set(target, declaration);
+                }
+            }
+        }
+        function collectEtsSymbols(file: SourceFile, statements: NodeArray<Statement>): void {
+            for (const statement of statements) {
+                if(statement.modifiers && findIndex(statement.modifiers, t => t.kind === SyntaxKind.ExportKeyword) !== -1) {

```

Comment:
I wrapped all the collection logic in a check for whether a statement has an `export` modifier. This was already on the one for `FunctionDeclaration`, but it seemed like desired behavior for all of them